docs/docs/guides/video_pipeline.md

```diff
@@ -2,26 +2,29 @@
 id: video_pipeline
 title: The Video Pipeline
 ---
-Frigate uses a sophisticated video pipeline that starts with the camera feeds and progressively applies transformations to them (e.g. decoding, motion detection, etc.).
+Frigate uses a sophisticated video pipeline that starts with the camera feed and progressively applies transformations to it (e.g. decoding, motion detection, etc.).
 
-This guide provides an overview to help users put the key Frigate concepts on a map.
+This guide provides an overview to help users understand some of the key Frigate concepts.
 
-### High level view of the video pipeline
+### Overview
```

Comment:
```suggestion
## Overview
```

Reply:
While this is conceptually right, I think it makes the navigation harded/weirder on the right bar. If we had more sections and sub sections I would agree, but I think that for this simple doc I would leave both sections as "equals"

Reply:
I am suggesting that all sections should be two, maybe I missed some

Reply:
the problem is that when future edits are made to the docs, the number of `#` is just incremented and then it leads to the section not having a side section at all.

Reply:
Ahhh I missed that... done